docs/DeveloperGuide.md

```diff
@@ -85,11 +85,11 @@ The `UI` component,
 
 ### Logic component
 
-**API** : [`Logic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
+**API** : [`InternshipLogic.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/logic/Logic.java)
```

Comment:
Should we change the link here to our repo?

Reply:
I was going to leave it for when I do my general edits to the DG, but yea, np, I'll put it in now.

Reply:
Np, I'll just do it now then